src/controller/CHIPDeviceController.cpp

```diff
@@ -1025,6 +1028,114 @@ CHIP_ERROR DeviceCommissioner::UnpairDevice(NodeId remoteDeviceId)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DeviceCommissioner::SendEchoRequest(NodeId remoteDeviceId)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    Messaging::SendFlags sendFlags;
+    System::PacketBufferHandle payloadBuf;
+    char * requestData = nullptr;
```

Comment:
Use RAII: ScopedBuffer.h

Reply:
I think we can just use buffer on stack for this small message content, no need to use dynamic memory.